types/eventsource/ts4.0/test/eventsource-dom.test.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference lib="DOM" />
+
+import EventSource = require("eventsource");
+
+const eventSource = new EventSource("http://foobar");
+let readyState: number = eventSource.readyState;
+let closedState: number = eventSource.CLOSED;
+closedState = EventSource.CLOSED;
+let connectingState: number = eventSource.CONNECTING;
+connectingState = EventSource.CONNECTING;
+let openState: number = eventSource.OPEN;
+openState = EventSource.OPEN;
+let url: string = eventSource.url;
+let withCredentials: boolean = eventSource.withCredentials;
+eventSource.onmessage = (event: MessageEvent) => {};
+eventSource.onerror = (event: MessageEvent) => {};
+eventSource.onopen = (event: MessageEvent) => {};
+eventSource.addEventListener = (type: string, listener: EventListener) => {};
+eventSource.dispatchEvent = (event: Event) => true;
+eventSource.removeEventListener = (type: string, listener: EventListener) => {};
+eventSource.close();
+
+// tslint:disable-next-line:no-relative-import-in-test Rule does not apply to nested typesVersions tests
+import EventSourcePolyfill = require("../lib/eventsource-polyfill");
```

Comment:
this is now a relative import due to a bad interaction between DT's structure and TS module resolution